### DISCUSSION TITLE
Flicker on refresh

### DISCUSSION THREAD

**Discussion by zejulio** [moononournation/Arduino_GFX#427] [Q&A]
Hello, I am using this library with a 2.0" display, and I am getting some flicker on refresh the display.
I usually do a gfx->fillScreen(BLACK); before writing the new text/draw, so I see the screen flicker...
What is the best way to do this so the flicker is not visible?

Thanks!

**Comment by moononournation** [comment]
avoid to use fillscreen while rewriting. you can set text background color instead.

**Comment by zejulio** [comment]
I have several screens how to clear all the screen before update? so I have sure that I dont have any remain from previous screen on the back?

**Reply by moononournation** [reply]
set text background color as black and use mono width font

**Reply by zejulio** [reply]
If I do not use a mono width font how can I do it?
I am using u8g2_font_helvR24_tf and it does not display properly, I believe it's not mono...
What do you suggest?

Thanks.

**Reply by moononournation** [reply]
try use the font type tail with "_h"

**Reply by zejulio** [reply]
where can I get a list with available fonts?

**Reply by moononournation** [reply]
I think it should at U8g2 page

**Reply by zejulio** [reply]
Can you give me an example of font name?

**Reply by zejulio** [reply]
How do I use u8g2_GetAscent and u8g2_GetDescent from your library?

Thanks!
